examples/torch/ppo_metaworld_mt50.py

```diff
@@ -0,0 +1,68 @@
+#!/usr/bin/env python3
+"""This is an example to train PPO on MT10 environment."""
+# pylint: disable=no-value-for-parameter
+import click
+from metaworld.benchmarks import MT50
```

Comment:
per CONTRIBUTING.md, this should be `import metaworld.benchmarks` and references should use `metaworld.benchmarks.MT50`